### ISSUE TITLE
rendered JSON type can be wrong for aggregate

### ISSUE THREAD

**Issue by guruofgentoo** [level12/webgrid#165]
Depending on the aggregation method, the returned column value can be numeric or string in the JSON response:
```
self.add_column(
    webgrid.Column(
        'Sum Returns String',
        sa.sql.func.sum(coalesce(self.SomeCTE.c.some_int_field)).label('some_int_field_sum'),
        has_subtotal=True,
    )
)
self.add_column(
    webgrid.Column(
        'Count Returns Numeric',
        sa.sql.func.count(coalesce(self.SomeCTE.c.some_int_field)).label('some_int_field_count'),
        has_subtotal=True,
    )
)
```

With the above, we can get JSON output like:
```
some_int_field_sum: "672",
some_int_field_count: 95
```

Note, this was produced by a Postgres db.

Workaround was to subclass the column and provide a `render_json` on it that cast to int.

**Comment by guruofgentoo** [comment]
This is due to the custom JSON encoder that currently only handles date types. Any other type that isn't serializable by default, just gets dumped to string. This one is a Decimal - we should update to handle that type.

**Comment by guruofgentoo** [comment]
Resolved in 0.4.12.
